src/index.html

```diff
@@ -24,6 +25,39 @@ <h1>Restaurant Menu</h1>
     </header>
     <main>
        <button class="test-comment">test popup comment</button> 
+
+       <ul class="menuSection" id="menuList">
+           <!-- <li class="menuItem">
+               <img src="./img/meal.svg">
+               <p>Name <i class='far fa-heart'></i></p>
+               <button>Comment</button>
+           </li>
+           <li class="menuItem">
+            <img src="./img/meal.svg">
+            <p>Name <i class='far fa-heart'></i></p>
+            <button>Comment</button>
+        </li>
+        <li class="menuItem">
+            <img src="./img/meal.svg">
+            <p>Name<i class='far fa-heart'></i></p>
+            <button>Comment</button>
+        </li>
+        <li class="menuItem">
+            <img src="./img/meal.svg">
+            <p>Name <i class='far fa-heart'></i></p>
+            <button>Comment</button>
+        </li>
+        <li class="menuItem">
+            <img src="./img/meal.svg">
+            <p>Name <i class='far fa-heart'></i></p>
+            <button>Comment</button>
+        </li>
+        <li class="menuItem">
+            <img src="./img/meal.svg">
+            <p>Name <i class='far fa-heart'></i></p>
+            <button>Comment</button>
+        </li> -->
```

Comment:
It would be better to remove this comment to make the code clean

Reply:
ok thank you for the review